Libraries/Image/RCTImageLoader.m

```diff
@@ -538,6 +492,48 @@ - (RCTImageLoaderCancellationBlock)decodeImageData:(NSData *)data
                                              scale:(CGFloat)scale
                                            clipped:(BOOL)clipped
                                         resizeMode:(RCTResizeMode)resizeMode
+                                          cacheKey:(NSString *)cacheKey
```

Comment:
Why do we need another variant of this method here? Please merge this together and update the callers.